_content/tour/spa/algorithms-data.article

```diff
@@ -1,24 +1,25 @@
-Data Structures
-This section provides data struct examples.
+Estructuras de datos
+Esta sección ofrece ejemplos de estructuras de datos.
 
 * Hash Map
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Ver el video]]
+- Si necesita asistencia financiera, utilice nuestro [[https://www.ardanlabs.com/scholarship/][Formulario de becas]]
 
-This sample program implements a basic hash table.
 
-- hashKey(key) returns a number between 0 to len(buckets)-1
+Este programa de muestra implementa una tabla hash básica.
 
-- We use a slice of entries as a bucket to handles cases where two or more keys
-  are hashed to the same bucket
+- La función hashKey(key) devuelve un número entre 0 y len(buckets)-1
 
-- See more at [[https://en.wikipedia.org/wiki/Hash_table][https://en.wikipedia.org/wiki/Hash_table]]
+- Utilizamos un slice de entries como bucket para manejar casos en los que dos o más claves
```

Comment:
What about using: cubo instead of bucket for this line and the ones below.

Reply:
I also think `entries` could be changed to `entradas`.

Reply:
Yep, I wasn't sure what to do there, as those are very common terms, but somehow makes sense to also translate them.  But then should we also the names in the diagram? Or other ones like key, value, etc ?

Reply:
Yeah as long as they are not language keywords I think we should translate. Otherwise someone learning might think a 'bucket' or 'key' are language specific words